frontend/src/pages/pipelines/global/modelCustomization/const.ts

```diff
@@ -3,15 +3,91 @@ export enum FineTunePageSections {
   BASE_MODEL = 'fine-tune-section-base-model',
   TEACHER_MODEL = 'fine-tune-section-teacher-model',
   JUDGE_MODEL = 'fine-tune-section-judge-model',
+  RUN_TYPE = 'fine-tune-section-run-type',
+  HYPERPARAMETERS = 'fun-tune-hyperparameters',
 }
 
 export const fineTunePageSectionTitles: Record<FineTunePageSections, string> = {
   [FineTunePageSections.PROJECT_DETAILS]: 'Project details',
   [FineTunePageSections.BASE_MODEL]: 'Base model',
   [FineTunePageSections.TEACHER_MODEL]: 'Teacher model',
   [FineTunePageSections.JUDGE_MODEL]: 'Judge model',
+  [FineTunePageSections.RUN_TYPE]: 'Run Type',
```

Comment:
`Type` should be lower case
```suggestion
  [FineTunePageSections.RUN_TYPE]: 'Run type',
```